client/src/components/Home.jsx

```diff
@@ -7,13 +7,20 @@ import MiniProfile from './MiniProfile.jsx';
 class Home extends React.Component {
   constructor (props) {
     super(props);
+    this.state = {
+      balance: 0
+    }
```

Comment:
Curious - Why initialize to $0 rather than to undefined? If the balance fails to load, do we want to display $0 balance?

Reply:
Makes sense. Will change it.